clouddriver-sql/src/main/resources/db/changelog/20190913-task-sagaids.yml

```diff
@@ -4,6 +4,7 @@ databaseChangeLog:
       dbms: postgresql
       remove: afterColumn
   - changeSet:
+      validCheckSum: 8:dac7716de0de84f4fd508d906d13bcaa
```

Comment:
I removed this line and the validCheckSum from 20181120-cats.yml and PostgresContainerTest still passed.  Is that expected?

Reply:
With liquibase 4.27.0 it passes. If you use 4.24.0, it fails.

Reply:
https://github.com/spinnaker/kork/pull/1184 hasn't been merged yet, so this PR still uses liquibase 4.24.0.

Reply:
When I hand-force the dependency to liquibase 4.27.0, PostgresContainerTest still passes for me with the validChecksum lines removed.

Reply:
My bad. It doesn't matter whether it's 4.24.0 or 4.27.0, the failure for new installation is not due to missing validchecksum (which causes failure to migrating spinnaker) but due to missing `removeChangeSetProperty` tag which was fixed through this commit - https://github.com/spinnaker/clouddriver/commit/db8897c38b5e676574cd6101df6d84514aa84a5d.

Reply:
Aaah ok.  I guess we haven't reached the platonic ideal of having a test that demonstrates the struggle with missing validCheckSum yet.  Testing a migration like that does sound fiddly, but might not even need a standalone clouddriver test like this.  It could be a "regular junit test" in clouddriver-sql.

Reply:
One way I can think of using test containers for migration testing is to add a script to pre-fill DatabaseChangeLog table so that when the test is run it mimics migration. But the script needs to be updated soon after every liquibase upgrade to keep it up-to date. 

I  am not sure of your proposal on regular junit test.

Reply:
I was thinking of adding an extra migration in clouddriver-sql/src/test/resources, but I'm not sure what it would take to be able to control when it's available.  As in, run a test that uses only the "real" migrations first and then adds the additional ones in a second step.

Reply:
But then both of them uses the same liquibase version. It won't be a true test, right?

Reply:
OK, that's another dimension then...and I guess a required one.  Pre-filling the database changelog table does seem like the way forward.